src/status_im/react_native/resources.cljs

```diff
@@ -49,6 +49,16 @@
    :graph              (js/require "../resources/images/ui/graph.png")
    :discover           (js/require "../resources/images/ui/discover.png")
    :no-contacts        (js/require "../resources/images/ui/no-contacts.png")
+   :ethereum-logo      (js/require "../resources/images/ui/Ethereum.png")
```

Comment:
these references can be removed 👍 we don't use this file anymore and try to avoid adding new code within status_im.* namespaces